pkg/cli/admin/inspect/pod.go

```diff
@@ -142,6 +155,19 @@ func (o *InspectOptions) gatherContainerRotatedLogFiles(destDir string, pod *cor
 				return
 			}
 
+			// make sure rotated logs honor --since and --since-time
+			// when set by the user.
+			if parts := strings.Split(fileName, "."); len(parts) >= 3 {
+				datePart := strings.Split(parts[2], "-")[0]
+				if rotatedTime, err := time.Parse("20060102", datePart); err == nil {
+					if rotatedTime.Before(requestedTime) {
+						return
```

Comment:
Do we need a high verbosity log about why users can not find some rotated logs, when they run must-gather?

Reply:
Good idea, added.